packages/cozy-harvest-lib/src/components/KonnectorConfiguration/KonnectorAccountTabs.jsx

```diff
@@ -87,83 +52,25 @@ const KonnectorAccountTabs = ({
 
             <TabPanels>
               <TabPanel name="data" className="u-pt-1-half u-pb-0">
-                <Stack>
-                  {isInMaintenance && (
-                    <div className="u-bg-paleGrey u-p-1">
-                      <KonnectorMaintenance
-                        maintenanceMessages={maintenanceMessages}
-                      />
-                    </div>
-                  )}
-                  {konnectorsModel.hasNewVersionAvailable(konnector) && (
-                    <KonnectorUpdateInfos
-                      konnector={konnector}
-                      isBlocking={hasTermsVersionMismatchError}
-                    />
-                  )}
-                  {shouldDisplayError && hasGenericError && (
-                    <TriggerErrorInfo error={error} konnector={konnector} />
-                  )}
-                  <LaunchTriggerCard
-                    initialTrigger={initialTrigger}
-                    disabled={isInMaintenance}
-                  />
-                  {has(initialTrigger, 'message.folder_to_save') && (
-                    <DocumentsLinkCard
-                      folderId={get(initialTrigger, 'message.folder_to_save')}
-                    />
-                  )}
-                </Stack>
+                <DataTab
```

Comment:
Nit: The `Tab` components represents the buttons to change tab, I think we should have either DataTabPanel or DataTabContent.